From 14f53ca0a3b47f5584172d2ac46630e3517e05f1 Mon Sep 17 00:00:00 2001
From: David Pacheco <dap@joyent.com>
Date: Tue, 27 Jun 2017 15:38:00 -0700
Subject: [PATCH] MANTA-3332 manta README out of date after MANTA-3288 Reviewed
 by: Trent Mick <trentm@gmail.com> Reviewed by: Kody A Kantor
 <kody.kantor@gmail.com> Approved by: Trent Mick <trentm@gmail.com> Approved
 by: Kody A Kantor <kody.kantor@gmail.com>

---
 README.md | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/README.md b/README.md
index a712a8f..c187829 100644
--- a/README.md
+++ b/README.md
@@ -42,8 +42,7 @@ the user documentation for details.
 * To see a detailed, real example of using Manta, check out [Kartlytics: Applying Big Data Analytics to Mario Kart](http://www.joyent.com/blog/introducing-kartlytics-mario-kart-64-analytics).
 
 * To learn about installing and operating your own Manta deployment, see the
-[Manta Operator's
-Guide](https://github.com/joyent/manta/blob/master/docs/operator-guide/index.md).
+[Manta Operator's Guide](https://joyent.github.io/manta/).
 
 * To understand Manta's architecture, see
 [Bringing Arbitrary Compute to Authoritative
-- 
2.21.0

